docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
             - "/var/run/docker.sock:/var/run/docker.sock"
         ports:
             - 8080:8080
-        image: functions/gateway:0.6.6-beta
+        image: functions/gateway:latest-dev
```

Comment:
This might be incorrect btw.

Reply:
For sure, for testing this is needed though (otherwise it will use `0.6.6-beta` which is on the remote registry)